### PR TITLE
Set up SASS and added colors and font styles

### PR DESCRIPTION
**Pull request by chloehwang**
Finished styling most of text.

### CODE REVIEW

File: src/stylesheets/base/_breakpoints.scss [hunk 1]
```diff
@@ -0,0 +1,77 @@
+.bp-mobile-only
```
Comment (ten1seven) [inline src/stylesheets/base/_breakpoints.scss]: This looks like Sass syntax.

File: src/stylesheets/config/_colors.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+// Variables
+$colors: (
+  lightPink: #f4dacf,
```
Comment (ten1seven) [inline src/stylesheets/config/_colors.scss]: I like to add comments to break up color groups for readability:

```scss
$colors: (
   // pinks
   lightPink: #f4dacf,
   slightlyDarkerPink: #f4d0c0,
   darkCoral: #d95d45,

   // grays
   lightGray: #f5f5f5,
   black: #000
);
```

File: src/stylesheets/base/_typography.scss [hunk 3]
```diff
@@ -1,3 +1,65 @@
...
+  &.-narrow-spacing {
+    letter-spacing: .15em;
+  }
+
+  &.-wide-spacing {
+    letter-spacing: .2em;
+  }
+
+  &.-san-serif {
+    @include font-family-geo-book
```
Comment (ten1seven) [inline src/stylesheets/base/_typography.scss]: Missing semicolon.

File: src/stylesheets/base/_typography.scss [hunk 4]
```diff
@@ -1,3 +1,65 @@
 body {
-  font-family: sans-serif;
+  @include font-family-lora;
+  font-size: $base-font-size;
+  line-height: $base-line-height;
+  color: black;
+}
+
+.text {
```
Comment (ten1seven) [inline src/stylesheets/base/_typography.scss]: Any harm in these modifiers being outside `.text`?

File: src/stylesheets/base/_typography.scss (hunk 3 above)
Comment (ten1seven) [inline src/stylesheets/base/_typography.scss]: Actually, looks like you're missing semicolons on a bunch of lines but don't want to comment on all of them. Though it's technically okay to omit the last semicolon, our team style has been to use them on all lines.

File: src/stylesheets/config/_typography.scss [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+
+// Variables
+$base-font-size   : 18px;
+$base-line-height : 32px;
+
+
+// Mixins
+@mixin font-family-lora {
+  font-family: 'Lora', "Cambria", "Times New Roman", "Times", serif;
```
Comment (ten1seven) [inline src/stylesheets/config/_typography.scss]: Mixed ' and ". Team style is to use ' whenever possible.

File: src/stylesheets/base/_breakpoints.scss (hunk 1 above)
Comment (chloehwang) [inline src/stylesheets/base/_breakpoints.scss]: yup, just brought over from another repo for reference

Comment (ten1seven) [inline src/stylesheets/base/_breakpoints.scss]: Does that mean you're not going to use it? If so, it should either not be committed or commented out so it's understood to not be part of the codebase.

**Pull request merged**